com.unity.ml-agents/Runtime/Sensors/ITypedSensor.cs

```diff
@@ -0,0 +1,28 @@
+namespace Unity.MLAgents.Sensors
+{
+
+    /// <summary>
+    /// The SensorType flag of the observation
+    /// </summary>
+    [System.Flags]
```

Comment:
If these are mutually exclusive, you shouldn't use `System.Flags`. If they're not mutually exclusive, the values should be powers of 2, e.g. `Observation = 1 << 0, Goal = 1 << 1, ...`